src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const { name,
+    age,
+    sex,
```

Comment:
move "name" to the next line)
```suggestion
export function Person({ person }) {
  const { 
    name,
    age,
    sex,
```